gitlab/v4/cli.py

```diff
@@ -357,7 +357,7 @@ def display_dict(d, padding):
                 id = getattr(obj, obj._id_attr)
                 print('%s: %s' % (obj._id_attr.replace('_', '-'), id))
             if hasattr(obj, '_short_print_attr'):
-                value = getattr(obj, obj._short_print_attr)
+                value = getattr(obj, obj._short_print_attr) or 'None'
```

Comment:
shouldn't `None` events just be skipped in the CLI?

```
target_title: None

target_title: None

target_title: None

target_title: None

target_title: None

target_title: None

target_title: None

target_title: None

target_title: None
```

Reply:
Actually, I don't know what these events are. It seems to me better to display them then to hode them. Because I consider github as a tool that helps me to show precisely what's going on. I it hides the results, it may hide an important clue to a problem I'm solving.

But I don't know how other parts of github behave. It's up to you to decide what you want it to do.

Reply:
The problem I see with this issue is that there is no way to have a usable output without displaying multiple attributes. For events that would be target_type and action_name for exemple

What we could do (to be discussed):

1. make the `display_dict` check for a `repr` method for the object. This method would return a key and a value to be displayed. We could use `__repr__` ou `__unicode__` for this maybe
2. if the method doesn't exist, fallback to the `_short_print_attr` attribute
3. if this attribute is None raise an error so we can fix the code and add the method from 1 (not sure about this step, it's not nice for end users).

Other idea: `_short_print_attr` could become a tuple, and we display multiple lines when needed